### DISCUSSION TITLE
setAttribute is not a function

### DISCUSSION THREAD

**Discussion by ikki-ai** [nuxt/nuxt#10327] [Questions (Nuxt 2)]
I got a bug that make my app unusable.
It seems to appear because of this function.
It is very hard to debug...
Someone could bring me some help ?

i'm on "nuxt-edge": "latest",
  "@nuxt/bridge": "npm:@nuxt/bridge-edge",
    "@nuxtjs/pwa": "^3.3.5",
thanks a lot for your help

function Rr(t, e) {
            var n = e.elm
              , data = e.data
              , r = t.data;
            if (!(o(data.staticClass) && o(data.class) && (o(r) || o(r.staticClass) && o(r.class)))) {
                var f = rr(e)
                  , l = n._transitionClasses;
                c(l) && (f = ir(f, ar(l))),
                f !== n._prevClass && (n.setAttribute("class", f),
                n._prevClass = f)
            }
        }

**Comment by ikki-ai** [comment]
the problem is very strange as it appears only on the manifest url when user is logged in on reload and only in prod (never in dev)

**Reply by davinma** [reply]
You can see [here](https://github.com/vuejs/vue/issues/10937), maybe it caused by SSR and `v-if`.
